sao.js

```diff
@@ -189,6 +197,13 @@ module.exports = {
     }
     */
 
+    // Format code according to eslint configuration
+    const linter = ctx.answers.eslint === 'standard' ? 'standard' : 'xo';
+    await spawn(`./node_modules/.bin/${linter}`, ['--fix'], {
```

Comment:
I'd switch this to `execa`.

Reply:
Is there a reason you're not just use `eslint` or `xo` for both fixes since standard is just a eslint config?

Reply:
@OmgImAlexis I was about to install `execa` but I realized I could just use the native `child_process`. But I can totally do that 👌🏻